recipe/meta.yaml

```
@@ -9,11 +9,12 @@ source:
   sha256: 5db25d4fd138013b563f9a3d1d87f7de7df1dac014fd4cccdfbb3435a5cff761
 
 build:
-  number: 1
-  skip: true  # [osx or win]
+  number: 2
+  noarch: generic
 
 requirements:
   run:
+    - __linux
```

This should ensure that the constructed `noarch` packages are generate for the intended OS & architecture (and are only installable there)

```suggestion
    - __linux               # [linux]
    - __win                 # [win]
    - __archspec * x86_64   # [x86_64]
    - __archspec * aarch64  # [aarch64]
    - __archspec * ppc64le  # [ppc64le]
```